plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/TestSqlServerConnectorTest.java

```diff
@@ -151,6 +155,22 @@ public void testInsertWriteBulkinessWithTimestamps(String timestampType)
         }
     }
 
+    @Test(dataProvider = "testTableNameDataProvider")
+    public void testCreateTableWithSpecialCharacterName(String tableName)
```

Comment:
testCreate -> testCreateAndDrop -- you test also that DROP works, and it's reasonable to test that